SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/CodeGenerator/Unreal/HelperFunctions.cs

```diff
@@ -0,0 +1,273 @@
+using Improbable.Codegen.Base;
+using Improbable.CodeGen.Base;
+using System.Collections.Generic;
+
+namespace Improbable.CodeGen.Unreal
+{
+    public static class HelperFunctions
+    {
+        public static string HeaderPath = "HelperFunctions.h";
```

Comment:
I suggest we use a more specific name as doing "`include "HelperFunctions.h"` on the cpp side in Unreal looks very non-descriptive. `ExternalSchemaHelperFunctions`?